### PR TITLE
Fix/aws ec2 resources

### PR DESCRIPTION
**Pull request by lgallard**
To avoid duplicate resource in AWS-EC2 Resource Model Source

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -11,7 +11,8 @@
   $manage_yum_repo    = $rundeck::manage_yum_repo,
   $package_ensure     = $rundeck::package_ensure,
   $package_source     = $rundeck::package_source,
-  $rdeck_home         = $rundeck::rdeck_home
+  $rdeck_home         = $rundeck::rdeck_home,
+  $manage_user        = $undeck::manage_user
```
Comment (juniorsysadmin) [inline manifests/install.pp]: typo ```$undeck```

File: manifests/config/global/framework.pp [hunk 2]
```diff
@@ -18,15 +18,15 @@
 
   if $ssl_enabled and $ssl_port == '' {
     $framework_config_port = { 'framework.server.port' => '4443' }
-    $framework_config_url = { 'framework.server.url' => "https://${::fqdn}:4443" }
+    $framework_config_url = { 'framework.server.url' => "http://${framework_config_base['framework.server.hostname']}:4443" }
```
Comment (juniorsysadmin) [inline manifests/config/global/framework.pp]: Why removal of https?

File: manifests/config/global/framework.pp [hunk 3]
```diff
@@ -18,15 +18,15 @@
...
   }
   elsif $ssl_enabled and $ssl_port != '' {
     $framework_config_port = { 'framework.server.port' => $ssl_port }
-    $framework_config_url = { 'framework.server.url' => "https://${::fqdn}:${ssl_port}" }
+    $framework_config_url = { 'framework.server.url' => "http://${framework_config_base['framework.server.hostname']}:${ssl_port}" }
```
Comment (juniorsysadmin) [inline manifests/config/global/framework.pp]: Why removal of https?

File: manifests/params.pp [hunk 4]
```diff
@@ -38,13 +38,13 @@
   $rdeck_home = '/var/lib/rundeck'
   $service_logs_dir = '/var/log/rundeck'
 
+  $manage_user = false
```
Comment (juniorsysadmin) [inline manifests/params.pp]: manage_user functionality should be done in a separate Pull Request

File: manifests/params.pp [hunk 5]
```diff
@@ -38,13 +38,13 @@
...
+
   $framework_config = {
     'framework.server.name'     => $::fqdn,
     'framework.server.hostname' => $::fqdn,
     'framework.server.port'     => '4440',
     'framework.server.url'      => "http://${::fqdn}:4440",
-    'framework.server.username' => 'admin',
```
Comment (juniorsysadmin) [inline manifests/params.pp]: Why the removal of these lines?

File: manifests/params.pp (hunk 4 above)
Comment (juniorsysadmin) [inline manifests/params.pp]: manage_user functionality should be done in a separate Pull Request

File: manifests/config/global/framework.pp (hunk 2 above)
Comment (juniorsysadmin) [inline manifests/config/global/framework.pp]: Why removal of https?

File: manifests/config/global/framework.pp (hunk 3 above)
Comment (juniorsysadmin) [inline manifests/config/global/framework.pp]: Why removal of https?

File: manifests/install.pp (hunk 1 above)
Comment (juniorsysadmin) [inline manifests/install.pp]: typo ```$undeck```

File: manifests/params.pp (hunk 5 above)
Comment (juniorsysadmin) [inline manifests/params.pp]: Why the removal of these lines?

File: manifests/params.pp (hunk 4 above)
Comment (lgallard) [inline manifests/params.pp]: Agree!

File: manifests/params.pp (hunk 5 above)
Comment (lgallard) [inline manifests/params.pp]: Agree!

File: manifests/install.pp (hunk 1 above)
Comment (lgallard) [inline manifests/install.pp]: Agree!

**Comment by lgallard** [comment]
@juniorsysadmin I tooked  these files from current master branch:

* manifests/config/global/framework.pp
* manifests/install.
*manifests/params.pp

Now the fix is only for the AWS EC2 resource

**Comment by bastelfreak** [comment]
Hi @lgallard, are you still interested in getting this merged? If so, could you please rebase against our latest master branch?
